go/libraries/doltcore/sqle/statspro/update.go

```diff
@@ -365,6 +370,21 @@ func (m mcvHeap) Counts() []uint64 {
 	return ret
 }
 
+func (m mcvHeap) Sort() {
+	sort.Slice(m, m.Less)
+}
+
+func (m *mcvHeap) Truncate(cutoff float64) {
+	start := m.Len()
+	for i, v := range *m {
+		if float64(v.cnt) >= cutoff {
+			start = i
```

Comment:
This should break out of the loop here, right?

Reply:
follow up bug fixes here https://github.com/dolthub/dolt/pull/8049